src/main/java/ch/njol/skript/conditions/CondGlowingText.java

```diff
@@ -0,0 +1,64 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.aliases.ItemType;
+import org.bukkit.block.Block;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.Sign;
+import org.bukkit.inventory.meta.BlockStateMeta;
+import org.bukkit.inventory.meta.ItemMeta;
+
+@Name("Has Glowing Text")
+@Description("Checks whether a sign (either a block or an item) has glowing text")
+@Examples("if target block has glowing text")
+@Since("INSERT VERSION")
+public class CondGlowingText extends PropertyCondition<Object> {
+
+	static {
+		register(CondGlowingText.class, PropertyType.HAVE, "glowing text", "blocks/itemtypes");
```

Comment:
You'll have to override this method https://github.com/SkriptLang/Skript/blob/eca99fef6bd5f3e504638454236ad54a1a67dd26/src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java#L141-L143

There are probably other syntax classes that suffer from this issue :(